src/js/actions/draganddrop.js

```diff
@@ -50,6 +50,16 @@ define(function (require, exports) {
     };
 
     /**
+    * Add many droppables at once
+    *
+    * @param {Iterable.List} list of droppable registration information
```

Comment:
Above is just a question, not a hard ask. Sometimes it's nice to overload "outer" functions so that they can take 1 or many, but then use many consistently internally. But, it's up to you.